deployment-local/add_credentials.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
```

Comment:
Please move all these resources in to deployment-local/src/main/resources folder.